rswag-specs/spec/rswag/specs/response_validator_spec.rb

```diff
@@ -73,6 +73,84 @@ module Specs
             it 'uses the referenced schema to validate the response body' do
               expect { call }.to raise_error(/Expected response body/)
             end
+
+            context 'nullable referenced schema' do
```

Comment:
We could move this to the context bellow `context 'openapi 3.0.1'` because this is actually not supported by OAS2

Reply:
sure I can move that down in the spec

Reply:
@sergioisidoro Ok I moved the new specs down to `context 'openapi 3.0.1' do` and updated the tests to use the `#/components/schemas/blog` syntax.